WebHostLib/templates/hostRoom.html

```diff
@@ -16,9 +16,9 @@
             This room has a <a href="{{ url_for("getTracker", tracker=room.tracker) }}">Multiworld Tracker</a> enabled.
             <br />
         {% endif %}
-        This room will be closed after {{ room.timeout//60//60 }} hours of inactivity. Should you wish to continue
-        later,
-        you can simply refresh this page and the server will be started again.<br>
+        The server for this room will be closed after {{ room.timeout//60//60 }} hours of inactivity.
```

Comment:
```suggestion
        The server for this room will be suspended after {{ room.timeout//60//60 }} hours of inactivity.
```
As per suggestion in discord.